README.md

```diff
@@ -1,17 +1,16 @@
 # Vault-Google Cloud Platform Common Library
 
-Utils for Vault/GCP integrations. Includes helpers for:
+Utilities for Vault and GCP integrations. Includes helpers for:
 - Parsing GCP credential JSON or finding default credentials
 - Various helpers for some GCP APIs
 
 This library was created to be shared by Vault-GCP integrations including the:
-- [GCP Auth Plugin](https://github.com/emilymye/vault-plugin-auth-gcp)
+- [GCP Auth Method](https://github.com/emilymye/vault-plugin-auth-gcp)
+- [GCP Secrets Engine](https://github.com/emilymye/vault-plugin-secrets-gcp)
```

Comment:
Do we want to link the repos from the `hashicorp/` org instead?

Reply:
Whoops! Done in https://github.com/hashicorp/go-gcp-common/pull/8/commits/dbeaedcf7363b486d2936133dc381d081f8fbea8 👍